pages/linux/ffsend.md

```diff
@@ -0,0 +1,23 @@
+# ffsend
+
+> Easy and securely share files from command line.
```

Comment:
s/Easy/Easily. Although there may be a better grammatical construction for this. @sbrl ?

Reply:
fixed